### DISCUSSION TITLE
如何调整论文正文里中英文的间距

### DISCUSSION THREAD

**Discussion by PhdBoLi** [sjtug/SJTUThesis#965] [提问]
![中英字符间距](https://github.com/sjtug/SJTUThesis/assets/33117973/6163fac2-8e10-4c24-97e4-1e32f9edef8c)
uu们，如题，请问如何全篇设置调整中英文字符间距

**Comment by LogCreative** [comment]
考虑提供改进后的目标效果。

**Comment by note286** [comment]
`xeCJK`手册Ctrl+F`CJKecglue`。
